test/acceptance/integration/Rules/Cards/card_create_rules.feature

```diff
@@ -201,7 +201,7 @@ Feature: Append a new rule in Perseo manager using cards from portal
     | test1300  | 34       | temper_23      | 12       | SendSmsMibAction | ${device_latitude}${device_longitude}${measure.value} | 123456789       |
     | test1300  | 34       | temper_23      | 0        | updateAttribute  | DANGER                                                | ALARM           |
 
-  @not_updated_card_interval_not_allowed @BUG_53
+  @not_updated_card_interval_not_allowed @BUG_53 @skip
```

Comment:
So you mean that whenever a "@skip" appears in the same line appears a "@BUG_XX" that is the cause of the skipping?